current_test.go

```diff
@@ -0,0 +1,48 @@
+package vaultrole
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/vault/api"
+)
+
+func Test_RoleExists(t *testing.T) {
+	raw := strings.TrimSpace(`
+{
+	"data": {
+		"keys": ["role-org-f1b776344f5a19dbc38243e915767ff2ef9234df"]
+	}
+}`)
+	secret := api.Secret{}
+	json.Unmarshal([]byte(raw), &secret)
+	testCases := []struct {
+		ID             string
+		Organizations  []string
+		ExpectedResult bool
+	}{
+		{
+			ID:             "123",
+			Organizations:  []string{"system:masters", "api"},
+			ExpectedResult: true,
+		},
```

Comment:
Lets have one or two other cases though. The raw data should then probably be part of the test case.